app/models/concerns/smooch_turnio.rb

```diff
@@ -97,7 +97,7 @@ def get_turnio_message_event(json)
     end
 
     def get_turnio_message_text(message)
-      message.dig('text', 'body') || message.dig('interactive', 'list_reply', 'title') || message.dig('interactive', 'button_reply', 'title') || ''
+      Bot::Smooch.get_capi_message_text(message)
```

Comment:
WhatsApp Cloud API and WhatsApp On-Prem API have the same payload for sent messages. So, avoid code duplication by using only one method for both.